pinot-common/src/main/java/org/apache/pinot/common/audit/AuditConfigManager.java

```diff
@@ -32,12 +40,10 @@
  * Self-registers with the provided cluster config provider.
  */
 @Singleton
-public final class AuditConfigManager {
-
+public final class AuditConfigManager implements ClusterConfigChangeListener {
```

Comment:
Can we follow the same pattern as in the server side? `DefaultClusterConfigChangeHandler` extends `ClusterConfigChangeListener` and is a generic framework that can be reused for other configs as well. Then have the Audit code register with that and get the config updates

There wasn't a need to add this for controller side yet, so had left it out at the time. But better to utilize a similar pattern IMO

Reply:
yeah sure, I'm fine with that as long as we get to the refactor relatively soon. thanks!